lib/update.nix

```diff
@@ -0,0 +1,47 @@
+{ pkgs, ... }:
```

Comment:
Not strictly necessary, but you could replace `pkgs` with the specific things you need from it (have a look at nix-watch.nix for an example)